Card/card.css

```diff
@@ -0,0 +1,248 @@
+@import url(../pageLayout.css);
+@import url(../Button/button.css);
+:root {
+    --card-lower-background-color: #f5f5f5;
+    --card-lower-border-color: #474747;
+    --image-container-color: white;
+    --card-border-color: black;
+    --card-strike-color: red;
+    --card-success-color: green;
```

Comment:
It is better to give the HEXCODES for these gneric colours also.
CAUSE: when giving colour-value og BLUE, each browser has its own variant , then your colour will be shown differently on various browsers